### PR TITLE
delete_network_peering called in post fail hook

### PR DESCRIPTION
**Pull request by rjagu1**
This PR(TEAM-8153) is about to fix Failure in mr_Test does not clean up the IBSm peering by calling delete_network_peering  in post fail hook.

- Related ticket: [TEAM-8153](https://jira.suse.com/browse/TEAM-8153)

- Verification run: http://openqaworker15.qa.suse.cz/tests/216615#
- Fail Scenario to check cleanup: http://openqaworker15.qa.suse.cz/tests/217042#step/mr_test/89

### CODE REVIEW

**Review by BillAnastasiadis — APPROVED** [review]
LGTM

File: tests/sles4sap/saptune/mr_test.pm [hunk 1]
```diff
@@ -103,6 +105,11 @@ sub post_fail_hook {
     if (get_var('PUBLIC_CLOUD_SLES4SAP')) {
         select_host_console(force => 1);
         my $args = OpenQA::Test::RunArgs->new();
+        record_info('Is the peering present?', join(' ', 'network_peering_present:', $self->{network_peering_present} // 'undefined'));
```
Comment (mpagot) [inline tests/sles4sap/saptune/mr_test.pm:108]: Other `record_info` printing the same variable are using `'CONTEXT LOG'` in place of `'Is the peering present?'`

**Review by mpagot — APPROVED** [review]
LGTM

**Review by mpagot — APPROVED** [review]

**Review by lilyeyes — COMMENTED** [review]
LGTM.

**Pull request merged**